### ISSUE TITLE
Orientation implementation

### ISSUE THREAD

**Issue by franciscocabrera23** [spark/AssetTracker#2]
Hi i have tried using the function getOrientation() which is private, i edit it and make it public, however there is no implementation on .cpp for it. Any ideas?

**Comment by technobly** [comment]
I'm guessing this is meant to give an orientation vector from the accelerometer.  Another way to hook this up might be to use the GPS heading response.

**Comment by franciscocabrera23** [comment]
Yes i need the orientation from North axis, for example 238 degrees from north.  Is this possible? Any help?

**Comment by technobly** [comment]
I would post your question on the community as feature request.  https://community.particle.io/

We can keep this open here as a thread to discuss what the getOrientation() function should provide.
